SU-HSDO/suhumsci
# Fix: Gradient hero right align bug

# READY FOR REVIEW

## Summary
When a gradient hero slide had more than one paragraph element within the body they are displaying in a row next to each other.

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. npm run test
2. Add a gradient hero slider to a flexible page.
3. Add text to the body area with two or more paragraph elements.
4. Those paragraphs should no longer float next to each other they should align to the right in a column above one another as they do when aligning left.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
